### DISCUSSION TITLE
Logging every second

### DISCUSSION THREAD

**Discussion by MikeBinstock** [basecamp/kamal#275] [Q&A] [answered]
after the app starts, I'am getting every second a logging for a healthcheck. Is that correct? Does make sense to log every second in the log files, even if the web app containers started and it's working well?

Tks!

**Comment by MikeBinstock** [comment]
<img width="1412" alt="Captura de pantalla 2023-05-07 a las 0 16 21" src="https://user-images.githubusercontent.com/764/236648602-2c64519c-3365-4ce1-aa6e-3513eb61b314.png">
this is an example of the logging

**Comment by dhh** [comment, marked as answer]
That is correct. This logging is done by the Rails app on receiving the healthcheck call which flows through a normal controller. So working as expected.
